extra/2-piping.js

```diff
@@ -16,26 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a+b;
 }
 
-function multiply() {
+function multiply(a, b) {
+  return a*b;
 
 }
 
-function format() {
-
+function format(num) {
+ return ("£" + num).toString();
```

Comment:
Nice thinking, but do we need to actually convert it to string explicitly? Check out the first answer on this stack overflow forum => https://stackoverflow.com/questions/40848551/how-does-adding-string-with-integer-work-in-javascript#:~:text=In%20JavaScript%2C%20the%20%2B%20operator%20is,string%20and%20concatenates%20both%20together.